quick-install/validate-tmpl.yaml

```diff
@@ -1,9 +1,12 @@
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   name: trow-validator
 webhooks:
   - name: validator.trow.io
+    sideEffects: None
+    admissionReviewVersions:
+      - v1beta1
```

Comment:
what does this mean?

Reply:
I found the information on them here: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/

They are required fields so added the simplest and recommended information. Realised I didn't add correct information for admissionReviewVersions (should be ["v1", "v1beta1"])